scala/lambdas/entity-event-generator-lambda/src/main/scala/uk/gov/nationalarchives/entityeventgenerator/Lambda.scala

```diff
@@ -65,45 +65,50 @@ class Lambda extends LambdaRunner[ScheduledEvent, Int, Config, Dependencies] {
   ): IO[Int] = {
     val ignoredEventTypes = IgnoredEventTypes.values.map(_.toString)
     for {
+      _ <- logger.info(s"Getting entities for triggered time $eventTriggeredDatetime")
```

Comment:
1. Maybe `s"Getting entities for event triggered time $eventTriggeredDatetime"`?
2. I'm not sure I understand this because it reads like we're getting entities with an updatedSince of `eventTriggeredDatetime`; should this message read something like `s"Getting entities at event triggered time $eventTriggeredDatetime"`, `s"Event triggered time: $eventTriggeredDatetime - getting entities"` or  `s"Getting entities - event triggered time: $eventTriggeredDatetime"`

Reply:
I wasn't thinking that hard about it. I wanted the event triggered time in there. I've changed it to `s"Getting entities - event triggered time: $eventTriggeredDatetime"`